<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScopeDialog</class>
 <widget class="QDialog" name="ScopeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">* {text-align: right}</string>
       </property>
       <property name="text">
        <string>scope description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="scopeText">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
     <item>
      <widget class="QPushButton" name="titleButton">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">* {text-align: right}</string>
       </property>
       <property name="text">
        <string>title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="scopeTitle"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
     <item>
      <widget class="QPushButton" name="classButton">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">* {text-align: right}</string>
       </property>
       <property name="text">
        <string>window class</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="scopeClass"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
     <item>
      <widget class="QPushButton" name="exeButton">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">* {text-align: right}</string>
       </property>
       <property name="text">
        <string>executable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="scopeExe"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QPushButton" name="a2ok_button">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="a2cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scopeText</tabstop>
  <tabstop>scopeTitle</tabstop>
  <tabstop>scopeClass</tabstop>
  <tabstop>scopeExe</tabstop>
  <tabstop>a2ok_button</tabstop>
  <tabstop>a2cancel_button</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>titleButton</tabstop>
  <tabstop>classButton</tabstop>
  <tabstop>exeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
